.github/workflows/integration.yml

```diff
@@ -11,17 +11,33 @@ on:
       - proxy-init/**
 
 jobs:
-  integration:
+  proxy-init-integration:
     timeout-minutes: 15
```

Comment:
would you mind adding `continue-on-error: true` here as well, I think that's a good fix.